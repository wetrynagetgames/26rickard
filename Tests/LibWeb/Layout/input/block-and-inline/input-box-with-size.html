<!DOCTYPE html>

<head></head>

<body>
  <input id="hello" size="12">
</body>

</html>