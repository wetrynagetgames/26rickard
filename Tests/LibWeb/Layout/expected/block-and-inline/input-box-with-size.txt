Viewport <#document> at (0,0) content-size 800x600 children: not-inline
  BlockContainer <html> at (0,0) content-size 800x37 [BFC] children: not-inline
    BlockContainer <(anonymous)> at (0,0) content-size 800x0 children: inline
      TextNode <#text>
    BlockContainer <body> at (8,8) content-size 784x21 children: inline
      frag 0 from BlockContainer start: 0, length: 0, rect: [11,10 115.125x17] baseline: 21
      TextNode <#text>
      BlockContainer <input#hello> at (11,10) content-size 115.125x17 inline-block [BFC] children: not-inline
        Box <div> at (11,10) content-size 115.125x17 flex-container(row) [FFC] children: not-inline
          BlockContainer <div> at (11,10) content-size 115.125x17 flex-item [BFC] children: inline
            TextNode <#text>
      TextNode <#text>

ViewportPaintable (Viewport<#document>) [0,0 800x600]
  PaintableWithLines (BlockContainer<HTML>) [0,0 800x37]
    PaintableWithLines (BlockContainer(anonymous)) [0,0 800x0]
    PaintableWithLines (BlockContainer<BODY>) [8,8 784x21]
      PaintableWithLines (BlockContainer<INPUT>#hello) [8,8 121.125x21]
        PaintableBox (Box<DIV>) [11,10 115.125x17]
          PaintableWithLines (BlockContainer<DIV>) [11,10 115.125x17]